scripts/install-windows-dev.ps1

```diff
@@ -58,7 +58,7 @@ function check_slack_binary_exist() {
       delay 0.2 "Heads up! A binary called ``$SLACK_CLI_NAME`` was found!"
       delay 0.3 "Now checking if it's the same Slack CLI..."
     }
-    & $SLACK_CLI_NAME _fingerprint | Tee-Object -Variable get_finger_print | Out-Null
+    & $SLACK_CLI_NAME --version | Tee-Object -Variable slack_cli_version | Out-Null
```

Comment:
👁️‍🗨️ note: Great findings from @ewanek1 hint that this `_fingerprint` might be causing CI to hang on a first installation?